src/App.tsx

```diff
@@ -11,6 +11,7 @@ import {
   isTest,
   shouldEnableGQLMockServer
 } from "./utils/getEnvironmentVariables";
+import TaskPage from "pages/Task";
```

Comment:
can we do named imports only please? For components and all other modules I prefer this approach because then the import supports autocomplete + is imported as the same thing everwhere